src/VisualStudio/Core/Def/ServicesVisualStudio.csproj

```diff
@@ -305,6 +313,9 @@
     <Reference Include="System.Xml.Linq" />
     <Reference Include="UIAutomationProvider" />
     <Reference Include="UIAutomationTypes" />
+    <Reference Include="VSLangProj2">
+      <HintPath>..\..\..\..\Binaries\VSLangProj2.dll</HintPath>
```

Comment:
This will not be checked-in as-is. I'm blocked on getting a nuget package with this binary (so I copied it locally to unblock progress).